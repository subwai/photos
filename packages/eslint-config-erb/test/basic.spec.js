const { CLIEngine } = require('eslint');

describe('basic', () => {
  it('should report semicolon errors', () => {
    const cli = new CLIEngine({
      useEslintrc: true,
    });
    const { results } = cli.executeOnFiles([require.resolve('./bad.ts')]);
    expect(
      results.map((result) =>
        result.messages.map((message) => ({
          ...message,
          filePath: '',
        }))
      )
    ).toMatchSnapshot();
  });
});
