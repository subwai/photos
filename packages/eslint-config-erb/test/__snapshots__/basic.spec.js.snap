// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic should report semicolon errors 1`] = `
Array [
  Array [
    Object {
      "column": 1,
      "endColumn": 12,
      "endLine": 2,
      "filePath": "",
      "line": 2,
      "message": "Unexpected console statement.",
      "messageId": "unexpected",
      "nodeType": "MemberExpression",
      "ruleId": "no-console",
      "severity": 1,
    },
    Object {
      "column": 7,
      "endColumn": 10,
      "endLine": 3,
      "filePath": "",
      "line": 3,
      "message": "'bar' is assigned a value but never used.",
      "messageId": "unusedVar",
      "nodeType": "Identifier",
      "ruleId": "@typescript-eslint/no-unused-vars",
      "severity": 1,
    },
  ],
]
`;
